app/views/welcome/index.html.erb

```diff
@@ -1,10 +1,11 @@
 <h1><%= @title_welcome %></h1>
 <h5><%= @descrip %></h5>
 <br>
-
-<% if current_user %>
-  <h1><%= "#{greetings.sample}, #{current_user.username.split("@").first}!" %></h1>
-<% else %>
-  <%= link_to "Login with Google",
-  "https://epeolatrys-rex.herokuapp.com/auth/google_oauth2" %>
-<% end %>
+<div class="ailgn-items-center">
```

Comment:
@lcpulzone @Caleb1991 not sure but wondering if this should be `align-items-center`?